### ISSUE TITLE
Produced compilation database is empty?

### ISSUE THREAD

**Issue by stepthom** [rizsotto/Bear#85]
I am using bear in a simple project, but the resulting compilation database has no content:

```
[dev@ubuntu:/dataware/irtools/testprograms/c/array_01 (11_doone)] $ cat source01.c 

int main()
{
    int x = 1;
    int y = 2;
    int array[10];

    if (x >= 10) 
    { 
        x = 0;
    }

    array[x] = 10;
    array[y] = 100;
    array[x+y] = 1000;

    return 0;
}

[dev@ubuntu:/dataware/irtools/testprograms/c/array_01 (11_doone)] $ cat Makefile 
CPP                 = clang
CPPFLAGS            = -pg -g -w -O0 -lm
SRCDIR              = .
SRCS                = $(wildcard $(SRCDIR)/*.c)
INCLUDE             = -I$(SRCDIR)

all:
	$(CPP) $(CPPFLAGS) $(INCLUDE) $(SRCS) -o a.out

clean: 
	rm -f a.out
	rm -rf out

[dev@ubuntu:/dataware/irtools/testprograms/c/array_01 (11_doone)] $ bear make
clang -pg -g -w -O0 -lm -I. ./source01.c -o a.out
[dev@ubuntu:/dataware/irtools/testprograms/c/array_01 (11_doone)] $ cat compile_commands.json 
[]
[dev@ubuntu:/dataware/irtools/testprograms/c/array_01 (11_doone)] $ bear --disable-filter make
clang -pg -g -w -O0 -lm -I. ./source01.c -o a.out
[dev@ubuntu:/dataware/irtools/testprograms/c/array_01 (11_doone)] $ cat compile_commands.json 
[
    {
        "command": [
            "clang", 
            "-pg", 
            "-g", 
            "-w", 
            "-O0", 
            "-lm", 
            "-I.", 
            "./source01.c", 
            "-o", 
            "a.out"
        ], 
        "directory": "/dataware/irtools/testprograms/c/array_01", 
        "function": "execvp", 
        "pid": "12504", 
        "ppid": "12503"
    }, 
    {
        "command": [
            "/usr/lib/llvm-3.5/bin/clang", 
            "-cc1", 
            "-triple", 
            "x86_64-pc-linux-gnu", 
            "-emit-obj", 
            "-mrelax-all", 
            "-disable-free", 
            "-disable-llvm-verifier", 
            "-main-file-name", 
            "source01.c", 
            "-mrelocation-model", 
            "static", 
            "-mdisable-fp-elim", 
            "-fmath-errno", 
            "-masm-verbose", 
            "-mconstructor-aliases", 
            "-munwind-tables", 
            "-fuse-init-array", 
            "-target-cpu", 
            "x86-64", 
            "-target-linker-version", 
            "2.24", 
            "-g", 
            "-dwarf-column-info", 
            "-resource-dir", 
            "/usr/lib/llvm-3.5/bin/../lib/clang/3.5.0", 
            "-I", 
            ".", 
            "-internal-isystem", 
            "/usr/local/include", 
            "-internal-isystem", 
            "/usr/lib/llvm-3.5/bin/../lib/clang/3.5.0/include", 
            "-internal-externc-isystem", 
            "/usr/bin/../lib/gcc/x86_64-linux-gnu/4.8/include", 
            "-internal-externc-isystem", 
            "/usr/include/x86_64-linux-gnu", 
            "-internal-externc-isystem", 
            "/include", 
            "-internal-externc-isystem", 
            "/usr/include", 
            "-O0", 
            "-w", 
            "-fdebug-compilation-dir", 
            "/dataware/irtools/testprograms/c/array_01", 
            "-ferror-limit", 
            "19", 
            "-fmessage-length", 
            "164", 
            "-pg", 
            "-mstackrealign", 
            "-fobjc-runtime=gcc", 
            "-fdiagnostics-show-option", 
            "-fcolor-diagnostics", 
            "-o", 
            "/tmp/source01-056183.o", 
            "-x", 
            "c", 
            "./source01.c"
        ], 
        "directory": "/dataware/irtools/testprograms/c/array_01", 
        "function": "posix_spawn", 
        "pid": "12504", 
        "ppid": "12503"
    }, 
    {
        "command": [
            "/usr/bin/ld", 
            "-z", 
            "relro", 
            "--hash-style=gnu", 
            "--build-id", 
            "--eh-frame-hdr", 
            "-m", 
            "elf_x86_64", 
            "-dynamic-linker", 
            "/lib64/ld-linux-x86-64.so.2", 
            "-o", 
            "a.out", 
            "/usr/bin/../lib/gcc/x86_64-linux-gnu/4.8/../../../x86_64-linux-gnu/gcrt1.o", 
            "/usr/bin/../lib/gcc/x86_64-linux-gnu/4.8/../../../x86_64-linux-gnu/crti.o", 
            "/usr/bin/../lib/gcc/x86_64-linux-gnu/4.8/crtbegin.o", 
            "-L/usr/bin/../lib/gcc/x86_64-linux-gnu/4.8", 
            "-L/usr/bin/../lib/gcc/x86_64-linux-gnu/4.8/../../../x86_64-linux-gnu", 
            "-L/lib/x86_64-linux-gnu", 
            "-L/lib/../lib64", 
            "-L/usr/lib/x86_64-linux-gnu", 
            "-L/usr/bin/../lib/gcc/x86_64-linux-gnu/4.8/../../..", 
            "-L/usr/lib/llvm-3.5/bin/../lib", 
            "-L/lib", 
            "-L/usr/lib", 
            "-lm", 
            "/tmp/source01-056183.o", 
            "-lgcc", 
            "--as-needed", 
            "-lgcc_s", 
            "--no-as-needed", 
            "-lc", 
            "-lgcc", 
            "--as-needed", 
            "-lgcc_s", 
            "--no-as-needed", 
            "/usr/bin/../lib/gcc/x86_64-linux-gnu/4.8/crtend.o", 
            "/usr/bin/../lib/gcc/x86_64-linux-gnu/4.8/../../../x86_64-linux-gnu/crtn.o"
        ], 
        "directory": "/dataware/irtools/testprograms/c/array_01", 
        "function": "posix_spawn", 
        "pid": "12504", 
        "ppid": "12503"
    }
][dev@ubuntu:/dataware/irtools/testprograms/c/array_01 (11_doone)] $ 
```
Any ideas?

**Comment by rizsotto** [comment]
@stepthom thanks for the report, can you send the output of `bear --version`?

**Comment by stepthom** [comment]
@rizsotto 

```
[dev@ubuntu:/dataware/sqlite (develop)] $ bear --version
bear 2.0.2
```

**Comment by rizsotto** [comment]
thanks, this issue supposed to be fixed by 2.0.3 version. (previous bug report was #80 ) please take a try with the new version, and if the bug is still there, please reopen this issue!

**Comment by stepthom** [comment]
![image](https://cloud.githubusercontent.com/assets/3842201/7814239/db59e364-038e-11e5-856c-1b0c0744b501.png)


@rizsotto It works! Thanks so much.
